src/sentry/sentry_apps/api/bases/sentryapps.py

```diff
@@ -178,7 +181,7 @@ def _get_organization_for_user(
         )
         if context is None or context.member is None:
             error_message = f"User does not belong to the '{organization_slug}' organization."
-            raise SentryAppIntegratorError(PermissionDenied(to_single_line_str(error_message)))
+            raise SentryAppError(message=to_single_line_str(error_message))
```

Comment:
`PermissionDenied` has status code 403

```suggestion
            raise SentryAppError(message=to_single_line_str(error_message), status_code=403)
```